### DISCUSSION TITLE
Bots

### DISCUSSION THREAD

**Discussion by 0xspade** [projectdiscovery/interactsh#117] [General]
Block bots, if we use notify and bots crawl it, it will be noisy notifs.

**Comment by ehsandeep** [comment]
@0xspade do you have more details to share? I'm not sure how interactsh is supposed to differentiate between real request and bot request?  also if you are using the self-hosted version of interactsh you can make it token protected to avoid random requests.

**Comment by MetzinAround** [comment]
Closing old discussion.
